src/App.js

```diff
@@ -4,53 +4,58 @@ import withRestrictedAccess from './components/hoc/withRestrictedAccess';
 import Dashboard from './components/pages/Dashboard/Dashboard';
 import Login from './components/pages/Login/Login';
 import CreateAccount from './components/pages/CreateAccount/CreateAccount';
-import { getUser } from "./utils/handlers/userHandlers";
-import { isLoggedIn, logout } from "./utils/handlers/authenticationHandlers";
+import { getUser } from './utils/handlers/authenticationHandlers';
+import { isLoggedIn, logout } from './utils/handlers/authenticationHandlers';
 
 function App() {
-
   const [user, changeUser] = useState(null);
 
   useEffect(() => {
     setUser(isLoggedIn());
   }, []);
 
-  const setUser = (bool) => {
-    if(bool){
+  const setUser = bool => {
+    if (bool) {
```

Comment:
I'm not 100% clear on what `bool` refers to in this case. Could you elaborate?

Reply:
I think this doesn't exist anymore. I refactored this part because it was a bit complicated, there were 3 functions involved to login, set the user, etc and now there is only the setState function and a hook